Zad4/src/com/zoo/animal/Animal.java

```diff
@@ -0,0 +1,54 @@
+package com.zoo.animal;
+
+public class Animal {
+    protected String name = "Animal";
+
+    protected void Say() {
```

Comment:
[ERROR] /Users/geoiva/Desktop/SberJava/students/alena_kochurova/nordic-it-java-q4/Zad4/src/com/zoo/animal/Animal.java:6:20: Method name 'Say' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'. [MethodName]